app/src/main/res/layout/profile_input_view.xml

```diff
@@ -4,8 +4,6 @@
   <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="28dp"
-    android:layout_marginEnd="28dp"
```

Comment:
This is affecting other files. When are we handling this. Are you creating another PR to fix all views.

Reply:
https://github.com/oppia/oppia-android/issues/1007

Reply:
Ack.